tests/test-contributors.php

```diff
@@ -678,7 +678,7 @@ public function test_contributorRoleNameChange() {
 	 * @group contributors
 	 */
 	public function test_handleImage() {
-		$src = $this->contributor->handleImage( 'https://dev.pressbooks.pub/app/plugins/pressbooks/assets/dist/images/default-book-cover.jpg' );
+		$src = $this->contributor->handleImage( 'https://dev.pressbooks.network/app/plugins/pressbooks/assets/dist/images/default-book-cover.jpg' );
```

Comment:
Test was failing because Ho Man and Christopher are migrating the dev.pressbooks.pub network so I changed URL to a more available one.